packages/components/Menu/src/MenuDivider/MenuDivider.types.ts

```diff
@@ -6,14 +6,20 @@ import type { IBackgroundColorTokens } from '@fluentui-react-native/tokens';
 
 export const menuDividerName = 'MenuDivider';
 
+export const MenuDividerInsetSizes = [0, 16, 56, 68, 72, 108] as const;
```

Comment:
Are these all the sizes for android divider? If yes, is there a type in divider that can be reused here?

Reply:
Correct same as Separator FURN Divider component. 
 
I used it before here ( https://github.com/microsoft/fluentui-react-native/pull/2715#discussion_r1160365923 ) but seems only for a type adding the entire divider dependencies wouldn't make sense. Even if tree shaking works but we want to manage the Menu<Item-Component> (such as MenuItemCheckbox. MenuItemRadio) as part of Menu.